dev/tools/examples_smoke_test.dart

```diff
@@ -165,6 +166,7 @@ void main() {
         expect(find.byType(WidgetsApp), findsOneWidget);
       } finally {
         ErrorWidget.builder = originalBuilder;
+        timeDilation = 1.0;
```

Comment:
Why is this reset here necessary? Nothing in the try seems to indicate that it would set the timeDilation?

Reply:
Because in the test app themselves's `main`, they set timeDilation, i.e. `${info.importName}.main();`.

I originally wanted to put the reset *there*, but realize that, we call a few pumps here, so if I reset the value in that main, the pump here will get dilation=1 which is wrong.

And cannot use tearDown as well, since that is too late (tearDown happens after the verification), so error will still happen